docs/user-guide/postinstall.md

```diff
@@ -0,0 +1,146 @@
+## Node installed with a DEB package
+Mysterium node can be installed using a DEB package for most of the Debian-like systems.
+
+[Prebuilt image](/user-guide/installation/#running-mysterium-node-on-a-raspberry-pi) for Raspberry Pi is using a node installed with the same DEB package.
+
+### Startup configuration
+The installed DEB package will be starting a node service automatically.
+
+You can check details using the following command:
+
+```
+sudo systemctl status mysterium-node.service
+```
+
+Systemd unit file, that describes service is located at the: `/lib/systemd/system/mysterium-node.service`.
+
+It should look similar to the following:
+
+```
+[Unit]
+Description=Server for decentralised VPN built on blockchain
+Documentation=https://mysterium.network/
+After=network-online.target
+
+[Service]
+User=mysterium-node
+Group=mysterium-node
+
+RuntimeDirectory=mysterium-node
+RuntimeDirectoryMode=0750
+LogsDirectory=mysterium-node
+
+EnvironmentFile=-/etc/default/mysterium-node
+ExecStart=/usr/bin/myst $CONF_DIR $RUN_DIR $DISCOVERY $BROKER $PROTO service --agreed-terms-and-conditions
+KillMode=process
+TimeoutStopSec=10
+SendSIGKILL=yes
+Restart=on-failure
+
+[Install]
+WantedBy=multi-user.target
+```
+
+The line starting with an `ExecStart=` describes how the process starts.
+You can change the line to configure the service by your need.
+
+List of available options can be found using a help commands: `myst --help` or `myst service --help`.
+
+For example, if you want to start only OpenVPN service on 1190 port and serve Mysterium verified consumer only, change the `ExecStart=` line to the following:
+
+```
+ExecStart=/usr/bin/myst $CONF_DIR $RUN_DIR $DISCOVERY $BROKER $PROTO service openvpn --openvpn.port=1190 --access-policy.list mysterium --agreed-terms-and-conditions
```

Comment:
Same here, all options can be configured here `/etc/default/mysterium-node`

Reply:
Fixed.